src/Config.php

```diff
@@ -1441,14 +1441,25 @@ function formatGlobalLayout(layout) {
         echo "<tr class='tab_bg_1'><th colspan='4' class='center b'>" . _n('Notification', 'Notifications', Session::getPluralNumber()) . "</th></tr>";
 
         echo "<tr class='tab_bg_2'>";
-        echo "<td><label for='dropdown_is_notif_enable_default$rand'>" . __('Enable notifications by default') . "</label></td><td>";
+        echo "<td><label for='dropdown_is_notif_enable_default$rand'>" . __('Enable notifications') . "</label></td><td>";
         Dropdown::showYesNo(
             'is_notif_enable_default',
             $data['is_notif_enable_default'],
             -1,
             ['rand' => $rand]
         );
-        echo "</td></tr>";
+        echo "</td>";
+        echo "<td colspan='2'>
+            <div class='alert alert-info d-flex align-items-center mb-0' role='alert'>
```

Comment:
as discussed together, an helper should be better than an additional td+div.alert

Reply:
done 
![image](https://github.com/glpi-project/glpi/assets/7335054/f9e814bd-18b9-4331-8a68-86b3f6fd451d)